### PR TITLE
Added shortcuts to all the matchers

### PR DESCRIPTION
**Pull request by Pazus**
Touches #283

### CODE REVIEW

**Comment by jgebal** [comment]
Will you be able to add the not_to shortcuts as well and update the documentation?

File: docs/userguide/expectations.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ To do that we use concept of expectation and a matcher to perform the check on t
 Example of unit test procedure body with a single expectation.
 ```sql
 begin
-  ut.expect( 'the tested value' ).to_( equal('the expected value') );
+  ut.expect( 'the tested value' ).to_equal('the expected value');
```
Comment (jgebal) [inline docs/userguide/expectations.md]: Could we keep both styles in the documentation for completeness?

File: source/expectations/ut_expectation_anydata.tps [hunk 2]
```diff
@@ -16,6 +16,8 @@ create or replace type ut_expectation_anydata under ut_expectation(
   limitations under the License.
   */
   overriding member procedure to_equal(self in ut_expectation_anydata, a_expected anydata, a_nulls_are_equal boolean := null),
```
Comment (jgebal) [inline source/expectations/ut_expectation_anydata.tps]: can you add `not_to_equal`

File: source/expectations/ut_expectation_date.tps [hunk 3]
```diff
@@ -16,6 +16,12 @@ create or replace type ut_expectation_date under ut_expectation(
   limitations under the License.
   */
   overriding member procedure to_equal(self in ut_expectation_date, a_expected date, a_nulls_are_equal boolean := null),
-  member procedure to_be_between(self in ut_expectation_date, a_lower_bound date, a_upper_bound date)
+  member procedure to_be_between(self in ut_expectation_date, a_lower_bound date, a_upper_bound date),
+  member procedure to_be_greater_or_equal(self in ut_expectation_date, a_expected date),
```
Comment (jgebal) [inline source/expectations/ut_expectation_date.tps]: can you add:
not_to_be_greater_or_equal
not_to_be_greater_than
not_to_be_less_than
not_to_be_less_or_equal

Comment (Pazus) [inline source/expectations/ut_expectation_date.tps]: Dont like the idea to add `not_to_be_less_than` cause its `to_be_greater_or_equal` etc.

Comment (Pazus) [inline source/expectations/ut_expectation_date.tps]: though implemented it)

**Pull request merged**